airframe-http/.jvm/src/main/scala/wvlet/airframe/http/codegen/client/ScalaHttpClientGenerator.scala

```diff
@@ -172,52 +172,95 @@ object ScalaJSClientGenerator extends HttpClientGenerator {
   override def defaultFileName: String  = "ServiceJSClient.scala"
   override def defaultClassName: String = "ServiceJSClient"
   override def generate(src: ClientSourceDef): String = {
-    def code =
+    def code: String =
       s"""${header(src.destPackageName)}
          |
          |import scala.concurrent.Future
          |import wvlet.airframe.surface.Surface
          |import wvlet.airframe.http.js.JSHttpClient
          |import wvlet.airframe.http.HttpMessage.Request
+         |import wvlet.airframe.rx.RxStream
          |
-         |${cls}""".stripMargin
+         |${jsClientCls}
+         |
+         |${rpcClientCls}""".stripMargin
```

Comment:
Both of them are created so we can choose, that's great.

Reply:
Right. When rendering DOM, Rx client is useful, but for calling RPC, which requires no DOM rendering, the existing ServiceJSClient is still necessary.